k8s-inception-demo.yml

```diff
@@ -1,15 +1,17 @@
 ---
-- hosts: gateway
+- hosts: localhost
   vars:
-    master_user: "{{ hostvars[groups['kube-master'][1]]['ansible_user'] }}"
-    master_ip: "{{ hostvars[groups['kube-master'][1]]['ip'] }}"
+    node_ip: "{{ hostvars[groups['kube-node'][0]]['ansible_host'] }}"
+    gateway_ip: "{{ hostvars['gateway']['ansible_host'] }}"
   tasks:
   - name: Kill existing forwards
-    shell: "ps aux | grep -v grep | grep 8888:localhost:31000 | awk '{print $2}' | sudo xargs --no-run-if-empty kill -9"
+    shell: "ps aux | grep -v grep | grep 8888:{{ node_ip }}:31000 | awk '{print $2}' | xargs --no-run-if-empty kill -9"
```

Comment:
You could probably simplify this with the `pkill` command, or at the least remove the `grep -v` by doing something like `grep [8]888:{{ node_ip }}:31000` so the match excludes `grep`.

Reply:
Nice tip re grep [8]888 ! I don't want to kill all ssh processes, only the one that is forwarding this port.